module/x/auction/keeper/test_common.go

```diff
@@ -623,20 +628,23 @@ func CreateTestEnv(t *testing.T) TestInput {
 			gravitykeeper.Hooks(),
 		),
 	)
+	autionkeeper := NewKeeper(auctionKey, getSubspace(paramsKeeper, auctiontypes.DefaultParamspace), marshaler, &bankKeeper, &accountKeeper, &distKeeper)
 
 	// set gravityIDs for batches and tx items, simulating genesis setup
 	gravitykeeper.SetLatestValsetNonce(ctx, 0)
 	// TODO: fix this
-	gravitykeeper.setLastObservedEventNonce(ctx, 0)
+	// gravitykeeper.setLastObservedEventNonce(ctx, 0)
```

Comment:
if we comment it out then valset will not be complete. Suggest that we just remove Valset from the test all together.